### PR TITLE
Fixing platform information for other direction finders.

### PR DESCRIPTION
**Pull request by ticcky**

### CODE REVIEW

File: alex/applications/PublicTransportInfoCS/hdc_policy.py [hunk 1]
```diff
@@ -376,33 +376,36 @@ def get_platform_res_da(self, ds, ludait, slots_being_requested,
...
-                        else:
-                            res_da.append(DialogueActItem('inform', 'platform',
-                                                      'none'))
-                            res_da.append(DialogueActItem('inform', 'track',
-                                                      'none'))
-                            res_da.append(DialogueActItem('inform', 'direction',
-                                                      platform_res.direction))
+                                                          platform_res.direction))
+                    except MethodNotSupported:
+                        res_da.append(DialogueActItem('inform', 'get_platform_not_supported'))
```
Comment (tuetschek) [inline alex/applications/PublicTransportInfoCS/hdc_policy.py]: I'd make it just general "not_supported" (with a general template "Tato funkce není dostupná").

**Comment by ticcky** [comment]
OK I fixed it and added a test.

File: alex/applications/PublicTransportInfoCS/test_hdc_policy.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+        user_input = DialogueActConfusionNetwork()
+        user_input.add(1.0, DialogueActItem(dai='info(task=find_platform)'))
+        user_input.add(1.0, DialogueActItem(dai='inform(from_stop=Praha)'))
+        user_input.add(1.0, DialogueActItem(dai='inform(to_stop=Brno)'))
+
+        state.update(user_input, system_input)
+        res = hdc_policy.get_da(state)
+
+        self.assert_('inform(get_platform_not_supported)' in res)
```
Comment (tuetschek) [inline alex/applications/PublicTransportInfoCS/test_hdc_policy.py]: `get_platform_not_supported` -> `not_supported` ;-).

File: alex/applications/PublicTransportInfoCS/nlg_templates.cfg [hunk 3]
```diff
@@ -1020,5 +1020,6 @@ templates = {
         "Vlak do stanice {direction} jede z nástupiště {platform}.",
     'inform(platform={platform})&inform(track={track})&inform(train_name={train_name})':
         "Vlak {train_name} jede z nástupiště {platform} kolej {track}.",
+    'inform(get_platform_not_supported)': "Bohužel tato funkce není dostupná."
```
Comment (tuetschek) [inline alex/applications/PublicTransportInfoCS/nlg_templates.cfg]: `get_platform_not_supported` -> `not_supported` ;-).

**Comment by ticcky** [comment]
Thanks! Fixed.

**Comment by tuetschek** [comment]
Thanks! This seems OK to me now (if you checked that runs :-) ).

**Pull request merged**